x/concentrated-liquidity/incentives_test.go

```diff
@@ -2884,6 +2991,11 @@ func (s *KeeperTestSuite) TestQueryAndClaimAllIncentives() {
 				}
 				s.Require().NoError(err)
 
+				communityPoolBalanceAfter := s.App.BankKeeper.GetAllBalances(s.Ctx, communityPoolAddress)
+				fmt.Println("communityPoolBalanceBefore", communityPoolBalanceBefore)
+				fmt.Println("communityPoolBalanceAfter", communityPoolBalanceAfter)
+				fmt.Println()
```

Comment:
Are these still useful?

Reply:
They are not, removed here https://github.com/osmosis-labs/osmosis/pull/5442/commits/0d3a36c6fce8d8c6a715abaf5c742a5143f4a581